Code/EnginePlugins/VisualScriptPlugin/Runtime/VisualScriptNodeUserData.h

```diff
@@ -74,7 +84,9 @@ namespace
     {
       EZ_SUCCEED_OR_RETURN(NodeUserData_Type::Serialize(nodeDesc, inout_stream, out_uiSize, out_uiAlignment));
 
-      inout_stream << nodeDesc.m_sTargetPropertyName;
+      const ezVariantArray& propertiesVar = nodeDesc.m_Value.Get<ezVariantArray>();
+
+      inout_stream << propertiesVar[0].Get<ezHashedString>();
```

Comment:
I guess I don't understand this code. Why are we serializing only the first variable here ?